src/storage/memoryStore.ts

```diff
@@ -73,8 +73,7 @@ export class MemoryStore<T extends ValidStoreType> implements SyncStore<T> {
         // TODO: more sane implementation
         try {
             return this.getItem(item) !== undefined;
-        }
-        catch {
+        } catch (e) {
```

Comment:
For some reason my test runner was breaking without (e). Not sure if it is my version of Node or something (v14.15).